homeassistant/components/upnp.py

```diff
@@ -98,10 +117,12 @@ def setup(hass, config):
         if internal == CONF_HASS:
             internal = internal_port
         try:
-            upnp.addportmapping(
-                external, 'TCP', host, internal, 'Home Assistant', '')
+            await service.add_port_mapping(internal, external, host, 'TCP',
+                                           desc='Home Assistant')
             registered.append(external)
-        except Exception:
+            _LOGGER.debug("external %s -> %s @ %s", external, internal, host)
+        except UpnpSoapError as error:
+            _LOGGER.error(error)
```

Comment:
Don't double log

Reply:
From Python documentation: "first matching except clause is triggered"

So there is no double log (tested)